<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>evaluate</class>
 <widget class="QDialog" name="evaluate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;方正清刻本悦宋简体&quot;;</string>
   </property>
   <property name="text">
    <string>请您对本次订单进行评价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="star1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="star2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="star3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="star4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="star5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border-image: url(:/Image/kuang8.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(96, 14, 89)
}
QPushButton:hover
{
border-image: url(:/Image/kuang9.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(148, 17, 137)
}
QPushButton:pressed
{
border-image: url(:/Image/kuang10.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(86, 11, 80)
}</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border-image: url(:/Image/kuang8.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(96, 14, 89)
}
QPushButton:hover
{
border-image: url(:/Image/kuang9.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(148, 17, 137)
}
QPushButton:pressed
{
border-image: url(:/Image/kuang10.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(86, 11, 80)
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
